WinApiFileHandleWrapper.cpp

```diff
@@ -0,0 +1,22 @@
+#include "WinApiFileHandleWrapper.h"
+
+WinApiFileHandleWrapper::WinApiFileHandleWrapper(LPCWSTR localFileName, 
+												 DWORD desiredAccess, 
+												 DWORD creationDisposition)
+	: m_file(CreateFile(localFileName, 
```

Comment:
It is recommended to work only with handle in this wrapper. U can open file outside.